### PR TITLE
Polymer

### PR DESCRIPTION
**Pull request by DonnaWuDongxia**
Two elements with demos: toggle-image-button and switch-image-button

### CODE REVIEW

**Comment by DonnaWuDongxia** [comment]
@leonhsl @halton @jondong Please have a look, they are common scaleable image buttons. We can use them in our demo pages. Thanks.

File: sample/src_polymer/elements/switch-image-button/demo/index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Demo for swith-image-button</title>
```
Comment (leonhsl) [inline sample/src_polymer/elements/switch-image-button/demo/index.html]: nits: swith --> switch

**Comment by leonhsl** [comment]
We'd better import the 2 new elements in our `elements.html` in this PR? Other than that, lgtm.

**Comment by DonnaWuDongxia** [comment]
@leonhsl updated. Thanks for reviewing.

**Comment by halton** [comment]
Thanks for PR and merge.

**Pull request merged**